Sources/Wendy/Docker/DockerCLI.swift

```diff
@@ -201,8 +201,8 @@ public struct DockerCLI: Sendable {
         return output.split(separator: "\n").map { String($0) }
     }
 
-    public func builderName(forPort port: Int) -> String {
-        return "wendy-builder-\(port)"
+    public var defaultBuilderName: String {
+        return "wendy-builder"
```

Comment:
what if we made this context aware? add a -docker-desktop or -orbstack ? that could get us an isolated builder per context. still multiple builders, but correctness /robustness is better.